main/fopen_wrappers.c

```diff
@@ -603,7 +603,9 @@ PHPAPI zend_string *php_resolve_path(const char *filename, size_t filename_lengt
 		const char *exec_fname = ZSTR_VAL(exec_filename);
 		size_t exec_fname_length = ZSTR_LEN(exec_filename);
 
-		while ((--exec_fname_length < SIZE_MAX) && !IS_SLASH(exec_fname[exec_fname_length]));
+		do {
```

Comment:
Can't this loop be rewritten in a cleaner way?
Like: check that the var is not 0 in the while loop condition along with the IS_SLASH, and modify the length in the while loop body. That seems more readable and cleaner. WDYT?

Reply:
matter of taste I would say, it does not change much things as readability IMHO but don t really object.

Reply:
Sure. But please at least put {} around the break (and break on a new line), as now the code doesn't really comply with our code style standard.